org.eclipse.winery.repository.rest/src/main/java/org/eclipse/winery/repository/rest/resources/MainResource.java

```diff
@@ -160,6 +161,11 @@ public YAMLParserResource yamlParser() {
         return new YAMLParserResource();
     }
 
+    @Path("threats/")
```

Comment:
Please remove the `/` since we want to get rid of the trailing slashes in the longterm.

Reply:
removed